wpilib/wpilib/drive/differentialdrive.py

```diff
@@ -70,13 +101,13 @@ class DifferentialDrive(RobotDriveBase):
     def __init__(self, leftMotor, rightMotor):
         """Constructor for DifferentialDrive.
 
-        .. note:: This class takes in a SpeedController per side. For two and four motor drivetrains,
-                  construct and pass in  :class:`.SpeedControllerGroup` instances as follows.
+        .. note:: To pass multiple motors per side, use a :class:`.SpeedControllerGroup`.
```

Comment:
While your docs are better than upstreams, I think we should keep upstreams

Reply:
This is what upstream has.

Reply:
Why can't I read?